### PR TITLE
Adjust security configuration for devops endpoint

### PR DESCRIPTION
**Pull request by axdotl**
Signed-off-by: Axel Köhler <axel.koehler@kiwigrid.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://developercertificate.org) signed
- [ ] Chart Version bumped (if the pr is an update to an existing chart)
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

File: charts/ditto-digital-twins/README.md [hunk 1]
```diff
@@ -77,3 +77,18 @@ gateway:
...
+```yaml
+gateway:
+  enableDummyAuth: false
+  devopsSecureStatus: true
+  extraEnv:
+    - name: DEVOPS_PASSWORD
+      value: foo
+    - name: STATUS_PASSWORD
+      value: bar
+``
```
Comment (monotek) [inline charts/ditto-digital-twins/README.md]: can we put passwords in secrets please?

Comment (axdotl) [reply charts/ditto-digital-twins/README.md]: This is just an example.

Comment (axdotl) [reply charts/ditto-digital-twins/README.md]: Let us improve this in an upcoming PR

Comment (monotek) [reply charts/ditto-digital-twins/README.md]: this is also wrong for an example, as it will be used this way.

**Pull request merged**